packages/core/platform/Makefile

```diff
@@ -2,16 +2,16 @@ NAMESPACE=cozy-system
 NAME=platform
 
 show:
-	helm template -n $(NAMESPACE) $(NAME) . --dry-run=server
+	helm template -n $(NAMESPACE) $(NAME) . --dry-run=server $$(kubectl api-versions | awk '{print "-a " $$1}')
```

Comment:
I don't like interpolations with

```
$(bash_command)
```
or 

```
`(bash_command)`
```

as they can lead to very difficult to debug errors. The better approach is to use xargs:

```bash
kubectl api-versions | awk '{print "-a " $$1}' | xargs -r -I{} helm template -n $(NAMESPACE) $(NAME) . --dry-run=server {}
```

in case if `awk` will return empty string, `xargs` won't run at all.

Reply:
like bonus - you don't need to use awkward syntax like double dollar

Reply:
yeah, but you still need it for awk :)

Reply:
I still have concerns about using `xargs` it might work diferent on various systems, awk is always the same.

Reply:
Not 100% accurate, you have `gawk` and `mawk` :)